lms/static/sass/search/_search.scss

```diff
@@ -0,0 +1,174 @@
+.search-bar,
+.course-index .search-bar,
+.dashboard .search-bar {
+
+  @include box-sizing(border-box);
+  position: relative;
+  padding: 5px;
+  font-family: $sans-serif;
+
+  form > div {
+    position: relative;
+  }
+
+  .search-field {
+    @include box-sizing(border-box);
+    @include border-radius(4px);
+    top: 5px;
+    width: 100%;
+    background: $white-t1;
+    font-family: $sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    letter-spacing: normal;
+    &.is-active {
+      background: $white;
+    }
+  }
+
+  .search-button, .cancel-button {
+    @include box-sizing(border-box);
+    display: block;
+    position: absolute;
+    top: 0;
+    @include right(12px);
+    height: 35px;
+    padding: 0;
+    color: #888;
+    font-size: 14px;
+    font-weight: normal;
+    line-height: 35px;
+    border: none;
+    box-shadow: none;
+    text-shadow: none;
+    text-transform: none;
+    letter-spacing: normal;
+    background: transparent;
+  }
+
+  .cancel-button {
+    display: none;
+  }
+
+}
+
+
+.search-results {
+
+  display: none;
+
+  .search-info {
+    padding-bottom: $baseline;
+    border-bottom: 4px solid $border-color-l4;
+    margin-bottom: $baseline;
+    h2 {
+      font-family: $sans-serif;
+    }
+    .search-count {
+      @include float(right);
+      color: $gray-l1;
+    }
+  }
+
+  .search-result-list {
+    padding: 0;
+  }
+
+  .search-results-item {
+    position: relative;
+    border-bottom: 1px solid $gray-l4;
+    list-style-type: none;
+    margin-bottom: $baseline;
+    padding-bottom: $baseline;
+    @include padding-right(140px);
+
+    .sri-excerpt {
```

Comment:
It isn't clear to me from this what sri stands for, I would perhaps use a simplification instead of an abbreviation for nested classes. (ex - .result-excerpt instead of .sri-excerpt)